script/generate_certificate.rb

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/ruby
+
+require "openssl"
+
+PRIV = OpenSSL::PKey::EC.new(
+  "\x30\x77\x02\x01\x01\x04\x20\x03\x84\x2a\xc7\xf4\xcd\xe3\x67\xde"\
+  "\xa0\x56\xc6\x4f\x7f\x3b\x15\xea\x7d\x4b\xc4\x83\xca\xc6\x97\x9f"\
+  "\x2a\x31\x93\xad\x57\x31\x09\xa0\x0a\x06\x08\x2a\x86\x48\xce\x3d"\
+  "\x03\x01\x07\xa1\x44\x03\x42\x00\x04\xf6\x9c\xab\x24\x14\x4b\xb4"\
+  "\xef\x87\xf7\x0f\x23\x1c\x5c\xd4\xf5\x78\x04\xac\xf8\xe0\xc6\xb2"\
+  "\xb3\xe3\x52\x18\x3d\x80\x39\x1f\x6b\xd2\x79\xd2\x6a\x4c\x83\x64"\
+  "\x74\xe6\xc2\xda\x23\x93\xff\xac\x1d\x50\x34\x6c\x5c\x23\x90\x65"\
+  "\x57\x93\x3e\xcb\x93\xff\x6e\xde\xd1"
+)
+
+# From "FIDO U2F Authenticator Transports Extension" spec.
+# X509 Extension OID for specifying supported transports.
+U2F_TRANSPORT_EXTENSION_OID = "1.3.6.1.4.1.45724.2.1.1"
+
+# From "FIDO U2F Authenticator Transports Extension" spec.
+# BIT STRING values for U2F_TRANSPORT_EXTENSION_OID.
+U2F_TRANSPORT_BLUETOOTH_RADIO            = 0b10000000
+U2F_TRANSPORT_BLUETOOTH_LOW_ENERGY_RADIO = 0b01000000
+U2F_TRANSPORT_USB                        = 0b00100000
+U2F_TRANSPORT_NFC                        = 0b00010000
+U2F_TRANSPORT_USB_INTERNAL               = 0b00001000
```

Comment:
It's a new value. One of the Google folks emailed me about it, asking for this change.